packages/core/types/index.js

```diff
@@ -1452,6 +1452,7 @@ export type ResolveResult = {|
   /** An absolute path to the file. */
   +filePath?: FilePath,
   +pipeline?: ?string,
+  +query?: QueryParameters,
```

Comment:
Can you document this? (That these are the query params the resolver didn't handle and should be forwarded to transformers)